src/main/java/com/openClassroomsProject/SafetyNetAlerts/model/Persons.java

```diff
@@ -0,0 +1,22 @@
+package com.openClassroomsProject.SafetyNetAlerts.model;
+
+import lombok.Data;
+import javax.persistence.*;
+
+@Data
+@Entity
+@Table(name="persons")
+public class Persons {
```

Comment:
This is a class -> Singular name, you are not returning a collection inside this class/

`Person`

Reply:
yes you're right